src/bezpath.rs

```diff
@@ -440,6 +440,63 @@ impl BezPath {
         }
         cbox.unwrap_or_default()
     }
+
+    /// Returns a new path with the winding direction of all subpaths reversed.
+    ///
+    /// If the path does not start with a move, returns `None`.
+    pub fn reverse_subpaths(&self) -> Option<BezPath> {
```

Comment:
If it doesn't start with a move we could also return the path, unchanged? I'm not sure how annoying the option is, here.

Reply:
As above, I think we should do some sort of validation step in fontc/write-fonts instead. I was trying to match behavior of `ReverseContourPen` but now I'm realizing that we're only checking validity when reversing contours which seems broken to me.